wpiunits/src/generate/main/java/Measure-interface.java.jinja

```diff
@@ -70,6 +70,18 @@ public interface {{ helpers['type_decl'](name) }} extends Measure<{{ helpers['mt
     return ({{ helpers['type_usage'](name) }}) unit().ofBaseUnits(baseUnitMagnitude() / divisor);
   }
 
+  /**
+  * {@InheritDoc}
+  *
+  * @deprecated use div instead. This was renamed for consistancy with other languages like Kotlin
```

Comment:
typo; "consist**a**ncy" should be "consist**e**ncy"

Or use maybe "... for compatibility with other languages..." instead?

Reply:
compatibility seems like it implies too much